SEOKMINJUN/Invaders-SDP
# Kan 44 integrate achievement with collection screen

## What
- Clean up the messy code.
- Modify it so that the clear count is accumulated.
- Change the color of the achievement name if it has been completed at least once.

## Why
- I liked the visual representation of completing achievements, so I made it so that if an achievement has been cleared at least once, the achievement name color changes.
- I think the modification to divide them into "complete" and "incomplete" and display them separately is good, so I kept it.

## How to Test
- Run the game.
- Clear the achievement.
- Go to the Collections tab and check.

## Screenshots
![image](https://github.com/user-attachments/assets/8c2c368d-d604-4bab-bfd8-79aac16c3890)
